ElectronicObserver/Window/Wpf/Log/LogView.xaml.cs

```diff
@@ -21,11 +26,12 @@ public LogView()
 		InitializeComponent();
 	}
 
-	private void ScrollViewer_ScrollChanged(object sender, ScrollChangedEventArgs e)
+	private void ListBox_ScrollChanged(object sender, ScrollChangedEventArgs e)
 	{
-		if (e.ExtentHeightChange != 0)
+		if (e.OriginalSource is ScrollViewer scrollViewer &&
+		Math.Abs(e.ExtentHeightChange) > 0.0)
 		{
-			ScrollViewer.ScrollToVerticalOffset(ScrollViewer.ExtentHeight);
+			scrollViewer.ScrollToBottom();
```

Comment:
When you set the font to `MS PGothic, 9pt`, the scroll to bottom behavior gets slightly odd. For some reason it leaves a large empty area at the bottom instead of scrolling down correctly.

Add this to LogViewModel constructor to have data available when you start debugging:
```cs
for (int i = 0; i < 1000; i++)
{
	LogList.Add("[2022/07/03 15:59:50][2] : #2「2」が遠征「02: Long Distance Training Voyage」へ出撃しました。");
	LogList.Add("[2022/07/03 16:13:21][2] : #3「3」が遠征「21: Northern Rat Transportation Strategy」から帰投しました。");
}
```

![image](https://user-images.githubusercontent.com/40002167/178089334-cae882e5-d4a8-4fdf-9188-532b53c21285.png)

I tried a few things but couldn't really find a way to fix it, might be a problem with the font.